### ISSUE TITLE
Integrate APIView into SDK Automation

### ISSUE THREAD

**Issue by weshaggard** [Azure/azure-sdk-tools#2341]
As part of improving the API review process we want to SDK Automation and PR process for the specs repo to include generating a APIView review session for the changes in the specs PR. 

High level requirements:
- Generate review for the relative changes in the PR
- Post a comment with a link to the review in the PR
- The review should already have diffs pre-computed so when the link is clicked it loads quickly (unlike the current link to OpenAPI hub diff which takes a while to load). 
- We should figure out how to integrate the existing OpenAPI links with APIView.

**Comment by dw511214992** [comment]
I think we only need to take care of 3rd requirement because others are supported. To support 3rd requirement, we need to ask apiview service to return the apiview link to apiview diff directly. @praveenkuttappan @maririos

**Comment by maririos** [comment]
FYI @ruowan

**Comment by weshaggard** [comment]
I think once @ruowan's work is done we will be close to this but I think we also need to get the baseline diffs added to ApiView as well and once that is done I will consider this done for swagger/openapi. I believe we already have another issue tracking cadl.

**Comment by maririos** [comment]
Thanks @weshaggard . Here is the summary:
- Pipeline work: https://github.com/Azure/azure-sdk-tools/issues/4441
- https://github.com/Azure/azure-sdk-tools/issues/4410

This is the issue for Cadl and PR support: https://github.com/Azure/azure-sdk-tools/issues/3991


Because work is tracked in specific issues I am going to close this one. If I missed something, feel free to re-open or file a speciic issue
